Maple2.Server.Game/Packets/ClubPacket.cs

```diff
@@ -0,0 +1,253 @@
+﻿using Maple2.Model.Enum;
+using Maple2.Model.Error;
+using Maple2.Model.Game.Club;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Constants;
+using Maple2.Server.Core.Packets;
+using Maple2.Tools.Extensions;
+
+namespace Maple2.Server.Game.Packets;
+
+public static class ClubPacket {
+    private enum Mode : byte {
+        UpdateClub = 0,
+        Establish = 1,
+        Create = 2,
+        DeleteStagedClub = 5,
+        Invited = 6,
+        Invite = 7,
+        AcceptInvite = 8,
+        InviteNotification = 9,
+        Leave = 10,
+        ChangeBuffNotification = 13,
+        StagedClubInviteReply = 15,
+        Disband = 16,
+        NotifyAcceptInvite = 17,
+        LeaveNotice = 18,
+        NotifyLogin = 19,
+        NotifyLogout = 20,
+        UpdateLeader = 21,
+        ChangeBuff = 22,
+        UpdateMemberMap = 23,
+        UpdateMember = 24,
+        Rename = 26,
+        UpdateMemberName = 27,
+        ErrorNotice = 29,
+        Join = 30,
+    }
+
+    public static ByteWriter Update(Club club) {
+        var pWriter = Packet.Of(SendOp.Club);
+        pWriter.Write(Mode.UpdateClub);
```

Comment:
These should all be `Write<Command>`

Reply:
It's for style, i don't like unparameterized writes because it's too easy to introduce bugs (e.g. type changes)